app/webpacker/css/darkswarm/hub_node.scss

```diff
@@ -267,4 +267,26 @@
       }
     }
   }
+
+  .ellipsed {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    width: 100%;
+  }
+
+  .no-wrap {
+    white-space: nowrap;
+  }
+
+  .flex {
+    display: flex;
+
+    &.align-center {
+      align-items: center;
+    }
+
+    &.justify-end {
+      justify-content: flex-end;
+    }
+  }
```

Comment:
Nice utilities. I think these would be good general classes to have available in other places too, and would suggest we move it to a generic file, perhaps named `utilities.scss`. Sound ok to you?

Reply:
I was afraid of name collision, but I did some changes, and this should be good ;)

Reply:
I did some commit to solve this. Please, let me know if it's ok for you!